src/main/java/com/ibm/as400/access/MessageQueue.java

```diff
@@ -66,7 +66,7 @@ The following is a list of attributes whose values are set on objects returned b
 
  @see  com.ibm.as400.access.QueuedMessage
  **/
-public class MessageQueue implements Serializable
+public class MessageQueue<T> implements Serializable
```

Comment:
The `<T>` isn't needed here since we aren't using a class-wide template type `T`